flask/app/groups.py

```diff
@@ -170,43 +187,52 @@ def create_group():
     except:
         abort(422, description="Invalid bucket name")
 
-    group_obj = models.Group(group_code=group_code, group_name=group_name)
+    group = models.Group(group_code=group_code, group_name=group_name)
 
     # Add users to the db if they were given
-    if request.json.get("users"):
+    have_users = (
```

Comment:
similar to the above, it seems like if the `users` field is passed in with the intention to add users to the group but the check fails, there is no message informing that the users have not been added to the group

Reply:
This updates the behaviour so we only attempt if `users` is of the correct format. Before it would have borked with a strange error while already creating the bucket, so the name wouldn't be usable without manually removing the bucket.